shark/src/main/java/shark/LeakTrace.kt

```diff
@@ -157,9 +157,13 @@ data class LeakTrace(
 
   internal fun fromV20(retainedHeapByteSize: Int?) = LeakTrace(
       gcRootType = elements!!.first().gcRootTypeFromV20(),
-      referencePath = elements.subList(
-          0, elements.lastIndex - 1
-      ).map { it.referencePathElementFromV20() },
+      referencePath = when {
+        elements.isEmpty() -> emptyList()
+        else -> elements
+                .subList(0, elements.lastIndex - 1)
+                .map { it.referencePathElementFromV20() }
+      }
+                      ,
```

Comment:
This comma looks lost! Can you please  put it next to curly on line 165?

Reply:
oh) Fixed!